### ISSUE TITLE
how to use deno docker in gitlab runner?

### ISSUE THREAD

**Issue by xlambda** [hayd/deno-docker#59]
this is the `.gitlab-ci.yml` file:
```
build:
  image:
    name: "hayd/debian-deno:latest"
  script:
    - echo hello
```
and gets the error:
```
Skipping Git submodules setup
Executing "step_script" stage of the job script
/bin/sh: 0: Can't open sh
```

If I add entrypoint like this:
```
build:
  image:
    name: "hayd/debian-deno:latest"
    entrypoint: ["/bin/sh"]
  script:
    - echo hello
```
still gets another error:
```
Skipping Git submodules setup
Executing "step_script" stage of the job script
error: Found argument 'sh' which wasn't expected, or isn't valid in this context

USAGE:
    deno [OPTIONS] [SUBCOMMAND]

For more information try --help
```

**Comment by hayd** [comment]
cc @DanielRamosAcosta  is this related/what you mentioned in #57 ?

**Comment by hork71** [comment]
what happens if you change the entrypoint?
entrypoint: [""]

**Comment by xlambda** [comment]
> 
> 
> what happens if you change the entrypoint?
> entrypoint: [""]

It works! thanks very much.

@hayd I've read the thread #57 but I have no idea if they are related since I'm new to gitlab ci and docker and deno.

**Comment by hayd** [comment]
should be fixed in 1.2.3
